aserver/worker.py

```
@@ -0,0 +1,78 @@
+import asyncio
+import nats
+from nats.aio.msg import Msg
+import logging
+import click
+from typing import List, Iterable
```

Ruff (0.8.2)

Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)